CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # Main (unreleased)
 
+- [ENHANCEMENT] Update postgres_exporter to a [branch of](https://github.com/grafana/postgres_exporter/tree/exporter-package-v0.10.0) v0.10.0
```

Comment:
Can you move this below the other enhancements? We try to keep the CHANGELOG sorted first by FEATURE > ENHANCEMENT > BUGFIX > CHANGE > DEPRECATION, and then by date per category.